pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -213,6 +216,23 @@ func validateImmutableFieldsCluster(new, old *Cluster) field.ErrorList {
 			field.Forbidden(specPath.Child("ControlPlaneConfiguration"), fmt.Sprintf("field is immutable %v", new.Spec.ControlPlaneConfiguration)))
 	}
 
+	if old.Spec.BundlesRef != nil && new.Spec.BundlesRef == nil {
+		allErrs = append(
+			allErrs,
+			field.Forbidden(specPath.Child("BundlesRef"), fmt.Sprintf("field cannot be removed after setting. Previous value %v", old.Spec.BundlesRef)))
+	}
+
+	standardBundleRef := regexp.MustCompile(`^bundle-[0-9]+$`)
```

Comment:
this prefix is incorrect - should be `bundles` (https://github.com/aws/eks-anywhere/blob/main/release/pkg/bundles/bundles.go#L33)